### PR TITLE
Fix startup scan

### PR DESCRIPTION
**Pull request by tylerwilliams**
Previously we were skipping some records when scanning the AC ranges on startup.

### CODE REVIEW

File: enterprise/server/backends/pebble_cache/pebble_cache.go [hunk 1]
```diff
@@ -971,13 +971,25 @@ func (e *partitionEvictor) computeSize() (int64, int64, int64, error) {
...
 
+	acPrefixRange := func(start, end []byte) ([]byte, []byte) {
+		left := make([]byte, 0)
+		left = append(left, e.acPrefix...)
+		left = append(left, start...)
+
+		right := make([]byte, 0)
+		right = append(right, e.acPrefix...)
+		right = append(right, end...)
+		return left, right
```
Comment (bduffany) [inline enterprise/server/backends/pebble_cache/pebble_cache.go:982]: nit: do you want to pre-compute the capacity for left and right?

File: enterprise/server/backends/pebble_cache/pebble_cache.go [hunk 2]
```diff
@@ -971,13 +971,25 @@ func (e *partitionEvictor) computeSize() (int64, int64, int64, error) {
...
+	}
+
 	// Start scanning the AC.
 	// AC keys look like /partitionID/ac/12312312313(crc-32)/digesthash
-	for i := 0; i < 100; i++ {
-		kr := append(e.acPrefix, []byte(fmt.Sprintf("%.2d", i))...)
-		start, end := keyRange(kr)
-		goScanRange(start, end)
+	for i := 10; i < 99; i++ {
```
Comment (bduffany) [inline enterprise/server/backends/pebble_cache/pebble_cache.go:987]: uber nit: maybe add a comment why it starts @ 10 here, and why 99 is handled as a special case

**Review by vadimberezniker — APPROVED** [review]

**Review by bduffany — APPROVED** [review]

File: enterprise/server/backends/pebble_cache/pebble_cache.go [hunk 3]
```diff
@@ -919,7 +919,7 @@ func (e *partitionEvictor) computeSizeInRange(start, end []byte) (int64, int64,
 		LowerBound: start,
 		UpperBound: end,
 	})
-	iter.SeekGE(start)
+	iter.SeekLT(start)
```
[the remark this answers is not in the document]
Comment (tylerwilliams) [reply enterprise/server/backends/pebble_cache/pebble_cache.go:922]: Still looking where I am going wrong, but when we use LT here it does not hit everything.

File: enterprise/server/backends/pebble_cache/pebble_cache.go (hunk 1 above)
Comment (tylerwilliams) [reply enterprise/server/backends/pebble_cache/pebble_cache.go]: Done.

File: enterprise/server/backends/pebble_cache/pebble_cache.go (hunk 2 above)
Comment (tylerwilliams) [reply enterprise/server/backends/pebble_cache/pebble_cache.go:988]: Done.

**Pull request merged**